test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.client.cs

```diff
@@ -23,7 +23,9 @@
 using Greet;
 using Grpc.Core;
 using Grpc.Net.Client;
+#if NET6_0_OR_GREATER
```

Comment:
Further down in this file we have a `#if !NETFRAMEWORK` directive.

https://github.com/open-telemetry/opentelemetry-dotnet/pull/3567/files#diff-9dafa45d9de8176d95436fabb3fcead50051f75d519bc1f4b99d782ad53962afL120-L126

I think it makes sense to standardize on one or the other. My vote for both of them to be `NET6_0_OR_GREATER`.